apps/docs/pages/guides/integrations/zuplo.mdx

```diff
@@ -0,0 +1,241 @@
+import Layout from '~/layouts/DefaultGuideLayout'
```

Comment:
I think it might just be an issue with the Vercel preview. It looks fine for me locally:
![zuplo-integration](https://user-images.githubusercontent.com/7026076/204416859-fe673d44-e3d1-4638-85ab-c3b52d4df839.png)